chronicle/release/change/change.go

```diff
@@ -25,7 +27,7 @@ type Reference struct {
 // ByChangeType returns the set of changes that match one of the given change types.
 func (s Changes) ByChangeType(types ...Type) (result Changes) {
 	for _, summary := range s {
-		if ContainsAny(types, summary.ChangeTypes) {
+		if ContainsAny(types, summary.ChangeTypes) || (len(summary.ChangeTypes) == 0 && types[0].Name == UnlabeledPRs) {
```

Comment:
this is accessing the first element of a list that is not guaranteed to be non-empty

Reply:
added a length check, above -- this function would never return anything if no types are provided and can bail